src/components/pages/roster-page/roster-page.tsx

```diff
@@ -90,6 +91,7 @@ export const RosterPage = () => {
   const [filtersOpen, setFiltersOpen] = useState(false);
   const [visibleColumnsMenuOpen, setVisibleColumnsMenuOpen] = useState(false);
   const [applyingFilters, setApplyingFilters] = useState(false);
+  const [errorDialogProps, setErrorDialogProps] = useState<ErrorDialogProps>({ open: false });
```

Comment:
I think this should be in ErrorBoundary (related to my comment in the ErrorBoundary component).

Reply:
Agreed